.buildkite/pipeline.yml

```
@@ -44,12 +44,13 @@ steps:
     if: *not_scheduled
     commands:
     - nix-build ci.nix -A build-ligo -o ./ligo-out/
-    - nix run -f ci.nix morley -c morley optimize --contract ./ligo-out/baseDAO.tz --output baseDAO.tz
+    - nix run -f ci.nix morley -c morley optimize --contract ./ligo-out/trivialDAO.tz --output trivialDAO.tz
+    - nix run -f ci.nix morley -c morley optimize --contract ./ligo-out/registryDAO.tz --output registryDAO.tz
+    - nix run -f ci.nix morley -c morley optimize --contract ./ligo-out/treasuryDAO.tz --output treasuryDAO.tz
     - cp ./ligo-out/trivialDAO_storage.tz trivialDAO_storage.tz
     - cp ./ligo-out/registryDAO_storage.tz registryDAO_storage.tz
     - cp ./ligo-out/treasuryDAO_storage.tz treasuryDAO_storage.tz
     artifact_paths:
-      - baseDAO.tz
       - trivialDAO_storage.tz
```

Why not give the compiled contracts as artifacts too?
```suggestion
      - trivialDAO.tz
      - registryDAO.tz
      - treasuryDAO.tz
      - trivialDAO_storage.tz
```